bin/libs/zos.ts

```diff
@@ -64,3 +66,35 @@ export function operatorCommand(command: string): { rc: number, out: string } {
   //we strip the '.' we added above
   return { rc: result.rc, out: result.out ? result.out.substring(0, result.out.length-1) : '' };
 }
+
+export function verifyGeneratedJcl(config:any): string {
+  const jcllib = config.zowe.setup.dataset.jcllib;
+  if (!jcllib) {
+    return undefined;
```

Comment:
This will return `undefined` only if `jcllib` is missing in `zowe.yaml` **AND** in `defaults.yaml`!
Let's suppose: `zowe.yaml` is missing `jcllib`.
Following code will check `IBMUSER.ZWEV2.SZWESAMP(ZWEIMVS)` and then execute `initGenerate()` without `dryRun` flag.
This will cause an attempt to copy `ZWEGENER` from nonexistent dataset.

Reply:
I will remove this from defaults.yaml
The point should be that users need to properly define which datasets content will go into, so if the value is undefined it is an indication they skipped a step.

Reply:
And L79: ` initGenerate.execute();` --> `initGenerate.execute(!!std.getenv('ZWE_CLI_PARAMETER_DRY_RUN'));`
It should respect `--dry-run` flag.

Reply:
Currently, dry run appears to mean "do no changes outside of jcllib"
It can be changed to mean "do no changes at all", but I'd rather save that for a different PR to prevent regressions. Can we postpone it?

Complications:

If generate doesn't run, dry run wont work for commands, because their dry run is to print out jcllib content that wouldn't exist.

`init mvs` and `init stc` currently modify their jcllib content because there is conditional behavior about member renaming that cannot be done in plain jcl.